src/Controls/src/Core/Page/Page.Mapper.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+namespace Microsoft.Maui.Controls;
+
+public partial class Page
+{
+	internal static new void RemapForControls()
+	{
+#if IOS
+		PageHandler.Mapper.ReplaceMapping<IContentView, IPageHandler>(PlatformConfiguration.iOSSpecific.Page.SafeAreaInsetsProperty.PropertyName, MapUpdateSafeAreaInsets);
```

Comment:
Should this be a command? The implementation is not actually doing anything with the value on the view? If the user sets it via the extension methods then nothing happens - and if that setting part is going away, then this should be a command - just like we do for the entry focus or the view frame.

Reply:
This seems fine. I think `Commands` are usually for initiating a request from the xplat to platform code `scrollto`, `remove`, etc.  This is a `mapper` property that associates with a `BindableProperty` so I think being inside a `mapper` is consistent with expectation. `BindableProprties` == `Mapper["Properties"]`

Reply:
But we also have the reverse with Focus and Frame. But what about the fact that this property is settable in Controls but we always override the value?

Reply:
Is it settable in controls? I think it's a readonly BP in `controls`

`Focus` I think makes sense as a `Command` because it's a request from the `xplat` code with args that's asking to change state.

`Frame` I feel like should have been a `mapper` property. The `frame` commandmapper even just calls a partial that has the signature of a mapper
```C#
static partial void MappingFrame(IViewHandler handler, IView view)
{
	handler.ToPlatform().UpdateAnchorX(view);
	handler.ToPlatform().UpdateAnchorY(view);
}
```

Reply:
Frame is only ever called from the platform layout I think? This happens after the layout has completed and the sets that property.

Reply:
Let's see,

The journey of `Frame`

Platform Layout is initiated => xplat.ArrangeOverride => Handler.PlatformArrange(Frame Calculated by Controls) => ViewHandler.PlatformArrangeHandler => platform code now translates this and calls `platformView.Layout` to do the actual layout => `viewHandler.Invoke(nameof(IView.Frame), frame)`

That last `invoke` call uses the original `frame` that's passed in which is the same `Frame` that's part of `IView.Frame` but isn't really used anywhere currently.

So, the point of `Frame` being a command seems to be so that initial execution doesn't happen.  `Frame` has a setter which I'm guessing doesn't generate the most expected behavior when you set it unless the control is overriding `OnSizeAllocated`. The command currently seems to primarily be for enabling side effects. Hey, we just called layout on the control, anything else you want to do? 

`SafeAreaInsets` is just some property we want to keep in sync with a platform property which we'd normally just do through an interface. 

So maybe `mapper` and `command` are both wrong for this and we lean into your other comment and kill two birds with one stone? 

Maybe we just add some internal interface for now to service this property that purely has a setter for the `SafeAreaInset` value.

the code in `MauiView` just becomes something like

if (View is ISafeAreaInsetAware saia)
    saia.Inset = this.SafeAreaInsets.ToThickness()

Reply:
Or we could just add it to `ISafeAreaView`

We're not obsoleting that interface and it seems like a viable property. 

That would then cause it to get set on `layout` and `page` though we wouldn't do anything with it on layout

Reply:
maybe we keep it internal on a new interface for now and we can always make it public on ISAV once we are semi-more-sure that we want it. I recall EZ had some opinions on what we should be doing for this. And then we don't have some arb property on Layout that we are not using and causing confusion. Once it is public it is hard to remove, but internal seems OK.

Reply:
Don't over-complicate this. `ISafeAreaView` is not long for this world, and the `SafeAreaInsets` platform-specific will be going away, too. The platform-specific is read-only, the only point of this PR is to make it work until we can get a better safe area API in place. Let's not add extra moving parts to this thing in the meantime.

Reply:
I guess I see the internal interface as being consistent with how we design these types of features and workflow. Who's to say when the good intentions of the future will come to fruition? Having the disconnect of a core `UpdateValue` without an `interface` partner just seems structurally confusing. 

An internal interface is just a different kind of throw-away code.

The API you've proposed internally would mostly match up with an interface here. Once the new name is decided this could/would just be renamed and anyone accessing it internally would just be changed.

Reply:
Plus it resolves @mattleibow 's comment here as well https://github.com/dotnet/maui/pull/15512#discussion_r1243645235

Reply:
@hartez @PureWeen @mattleibow - Updated now! I added the interface with just that one property and use that as the name for the property in the other discussion: https://github.com/dotnet/maui/pull/15512#discussion_r1243645235

Reply:
Okay, updated again to solely use the interface to set the property and got rid of the mapper entirely